tests/modules/epang/place/test.yml

```diff
@@ -0,0 +1,13 @@
+- name: epang_place test_epang_place
+  command: nextflow run ./tests/modules/epang/place -entry test_epang_place -c ./tests/config/nextflow.config  -c ./tests/modules/epang/place/nextflow.config
+  tags:
+    - epang
+    - epang/place
+  files:
+    - path: output/epang/test.epa_info.log
+      contains:
+        - "INFO 3 Sequences done"
+    - path: output/epang/test.epa_result.jplace
```

Comment:
No checksum? So the output isnt reproducible? (Just making sure)

Reply:
According to the test generator it's not, which  is a bit funny as it's not gzipped. Haven't looked further into this.